web-server/app/views/resultFrame.scala.html

```diff
@@ -20,7 +20,7 @@
 )
 
 @regex = @{
-    RegexConstructor(query = query, space = space, insensitive = insensitive, precise = precise)
+    RegexConstructor(query = query, space = space, insensitive = insensitive, precise = precise).r
```

Comment:
Please define a plain String here. Remove `.toString` invocation on 49 row, and invoke `.r` on 62 row.